src/app/common/abstract-entity.service.ts

```diff
@@ -0,0 +1,131 @@
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+
+import { of, Observable } from 'rxjs';
+import { catchError, map, tap } from 'rxjs/operators';
+
+import { NULL_PAGING_RESULTS, PagingOptions, PagingResults } from './paging.module';
+import { SystemAlertService } from './system-alert/system-alert.service';
+
+export enum ServiceMethod {
+	create = 'create',
+	get = 'get',
+	update = 'update',
+	search = 'search',
+	remove = 'remove'
+}
+
+export abstract class AbstractEntityService<T extends { _id: string }> {
```

Comment:
@jrassa - was this class added intentionally? Or did you just want to update the `asy-table-data-source.ts` file?

It looks good to me overall, and a great candidate for test coverage with a concrete test service!

Reply:
I did not intend to include this class in this PR.  Cleaning it up now.